day_7/high_level.md

```diff
@@ -0,0 +1,56 @@
+# How I would solve Ceasar Cipher problem
+1. Assuming we made an object CeasarCipher object of a class, I'll make a class named CeasarCipher
+2. Learn about ASCII values
+3. Assuming encode is a function, i will create an encode function with parameter str
+4. Using for loop I will make my code read every character and convert character(capital) to its decimal value
+5. At the end I will use the given code to shift character by five
+
+### Solution to Ceasar Cipher problem
```

Comment:
Did you find this solution somewhere or create on your own?

Reply:
@ericweissman I'm a CIS student and I've been teaching myself Java since the quarantine started. I knew about ASCII and if-else statements and after I figure out what Ceasar Cipher is, it was pretty easy to solve it.